manifests/attribute_filter.pp

```diff
@@ -7,10 +7,11 @@
 
   # Create the attribute-filter.xml configuration file.
   concat { 'attribute-filter.xml':
-    path  => "${shibboleth_idp::shib_install_base}/conf/attribute-filter.xml",
-    owner => $shibboleth_idp::shib_user,
-    group => $shibboleth_idp::shib_group,
-    mode  => '0600',
+    path   => "${shibboleth_idp::shib_install_base}/conf/attribute-filter.xml",
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0600',
+    notify => Class['shibboleth_idp::service'],
```

Comment:
Have you tested this? Does it work as expected? So if only only fragment of the file is updates the notify works? An alternative might be to make the whole class notify the service class in the init, but this seems like it could work too. An example from the apache module etc, is also good evidence that it works...